File: solutions/php/1/1.php
<?php

declare(strict_types=1);
error_reporting(E_ALL);

$sum = 0;

for ($num = 1; $num < 1000; $num++)
{
    if ($num % 3 === 0 || $num % 5 === 0)
    {
        $sum += $num;
    }
}

print("$sum\n");

File: solutions/go/1/1.go
package main

import (
	"fmt"
)

func main() {
	var sum int = 0

	for num := 1; num < 1000; num++ {
		if num%3 == 0 || num%5 == 0 {
			sum += num
		}
	}

	fmt.Println(sum)
}
